charts/prometheus-node-exporter/values.yaml

```diff
@@ -68,7 +72,8 @@ securityContext:
   runAsNonRoot: true
   runAsUser: 65534
 
-containerSecurityContext: {}
+containerSecurityContext:
```

Comment:
please revert

Reply:
Done, sorry I saved and auto-formatted it but thought I'd fully reverted it all.